src/pylon/deploy/pylon-config/location.conf.template

```diff
@@ -80,6 +80,11 @@ location ~ ^/alert-manager(.*)$ {
   proxy_pass {{ALERT_MANGER_URI}}/alert-manager$1$is_args$args;
 }
 
+# log-manager server
+location ~ ^/log-manager/([^/]+):(\d+)/(.*)$ {
+  proxy_pass http://$1:$2/$3$is_args$args;
```

Comment:
because it uses ip of each node, you'd better use a central ip and make the node ip in the path
otherwise, you may not write a `sub_filter` to match each node's ip

Reply:
e.g. `node-ip:log-port/log-path` -> `pylon-address/log-manager/node-ip/log-path`